.github/workflows/smi-conformance.yml

```diff
@@ -0,0 +1,19 @@
+name: SMI Conformance with Meshery
+on:
+  workflow_run:
+    workflows: ["Release Helm Chart"]
+    types:
+      - completed
+
+jobs:
+  smi-conformance:
+    name: SMI Conformance
+    runs-on: ubuntu-latest
+    steps:
+
+      - name: SMI conformance tests
+        uses: layer5io/mesheryctl-smi-conformance-action@master
+        with:
+          provider_token: ${{ secrets.MESHERY_PROVIDER_TOKEN }}
```

Comment:
Feel free to reach out to me to have this updated.

Reply:
Sure, since this is sensitive, we could discuss on Slack. You can find me on the OSM/CNCF Slack channel as **Navendu Pottekkat**.